### ISSUE TITLE
remove fliers from boxenplot

### ISSUE THREAD

**Issue by arnavlavan** [mwaskom/seaborn#1829]
I tried removing fliers from boxenplot using plt.scatter keywords (marker='') and I got this error:

>  File "C:\Users\OWNER\AppData\Local\Programs\Python\Python37-32\lib\site-packages\seaborn\categorical.py", line 1969, in _lvplot
>     marker='d', c=mpl.colors.rgb2hex(color), **kws)
> TypeError: scatter() got multiple values for keyword argument 'marker'

Thanks, Adam

**Comment by MaozGelbart** [comment]
My expectation about the correct way of removing fliers is setting ``outlier_prop=0`` but it retains the extreme points. Example code:
```python
import seaborn as sns
tips = sns.load_dataset("tips")
ax = sns.boxenplot(x=tips["total_bill"], outlier_prop=0)
```  
No exception thrown, but outliers are plotted.

**Comment by mwaskom** [comment]
Closed by #2010
